@xen-orchestra/lite/src/App/Pools/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Map } from 'immutable'
+import { Route, Switch } from 'react-router'
+import { withState } from 'reaclette'
+
+import PoolUpdates from './PoolUpdates'
+
+import { ObjectsByType, Pool } from '../../libs/xapi'
+
+interface ParentState {
+  objectsFetched: boolean
```

Comment:
Normally, we won't need to declare it a second time, as long as it's in the parent state of App.